pol-core/bscript/compiler/ast/FunctionExpression.cpp

```diff
@@ -0,0 +1,26 @@
+#include "FunctionExpression.h"
+
+#include <fmt/format.h>
+
+#include "bscript/compiler/ast/NodeVisitor.h"
+#include "bscript/compiler/model/FunctionLink.h"
+
+namespace Pol::Bscript::Compiler
+{
+FunctionExpression::FunctionExpression( const SourceLocation& source_location,
+                                        std::shared_ptr<FunctionLink> function_link )
+    : Value( source_location ), function_link( std::move( function_link ) )
+{
+}
+
+void FunctionExpression::accept( NodeVisitor& visitor )
+{
+  visitor.visit_function_expression( *this );
+}
+
+void FunctionExpression::describe_to( std::string& w ) const
+{
+  fmt::format_to( std::back_inserter( w ), "boolean-value({})", true );
```

Comment:
That looks like c&p error :)

Reply:
Yeah it's a placeholder until I figure out how to properly describe it 👍